crates/matrix-sdk/CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased] - ReleaseDate
 
+### Refactor
+
+- [**breaking**] Move the optional `RequestConfig` argument of the
+  `Client::send()` method to the `with_request_config()` builder method. You
+  should call `Client::send(request).with_request_config(request_config).awat`
```

Comment:
```suggestion
  should call `Client::send(request).with_request_config(request_config).await`
```

Reply:
Oh, that's a nice typo.

Reply:
https://github.com/matrix-org/matrix-rust-sdk/pull/4446